internal/transactions/add-sig.go

```diff
@@ -0,0 +1,189 @@
+/*
+ * Flow CLI
+ *
+ * Copyright Flow Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package transactions
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+
+	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/crypto"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flowkit/v2"
+	"github.com/onflow/flowkit/v2/accounts"
+	"github.com/onflow/flowkit/v2/config"
+	"github.com/onflow/flowkit/v2/output"
+	"github.com/onflow/flowkit/v2/transactions"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/internal/prompt"
+)
+
+type flagsAddSig struct {
+	Include  []string `default:"" flag:"include" info:"Fields to include in the output. Valid values: signatures, code, payload."`
+	KeyIndex string   `default:"0" flag:"key-index" info:"Account key index"`
+}
+
+var addSigFlags = flagsAddSig{}
+
+var addSigCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "add-sig <built transaction filename> <address> <signature>",
```

Comment:
Can we make the full use be `add-signature` and then have a shorthand for `add-sig`?